kwonhayeon/sql/create_tables.sql

```diff
@@ -0,0 +1,19 @@
+-- 새 데이터베이스 생성 및 선택
+CREATE DATABASE library_db;
+USE library_db;
+
+-- 사용자 테이블
+CREATE TABLE user (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL, -- 이름은 필수 입력 항목이므로 NOT NULL 지정
+    email VARCHAR(100) NOT NULL -- 이메일은 중복 되면 안 되므로 UNIQUE 지정
```

Comment:
설명과 달리 UNIQUE가 지정되어 있지 않은 것 같아요!

```suggestion
    email VARCHAR(100) NOT NULL UNIQUE
```